httpserver/http_connection.cpp

```diff
@@ -546,6 +543,26 @@ bool connection_t::parse_header(const char* first, Args... args) {
     return parse_header(first) && parse_header(args...);
 }
 
+json connection_t::get_snodes_json_by_pk(const std::string& pubKey) {
```

Comment:
You can make this function const` as well, after constifying `service_node::get_snodes_by_pk` :)

Reply:
Correct